### PR TITLE
Add and standardize search close button

### PR DESCRIPTION
**Pull request by samuelfullerthomas**
screen shot 2016-11-06 at 3 35 50 pm

Add and standardize a close button to search inputs. Addresses #929 & #935

### CODE REVIEW

**Comment by samuelfullerthomas** [comment]
@jasonLaster

File: public/js/components/EditorSearchBar.js [hunk 1]
```diff
@@ -55,12 +56,15 @@ const EditorSearchBar = React.createClass({
   },
 
   onEscape(shortcut, e) {
+    this.closeSearch(e);
+  },
+  closeSearch(ev) {
```
Comment (jasonLaster) [inline public/js/components/EditorSearchBar.js]: needs a space

File: public/js/components/CloseButton.js [hunk 2]
```diff
@@ -4,13 +4,13 @@ const Svg = require("./utils/Svg");
 
 require("./CloseButton.css");
 
-function CloseButton({ handleClick }) {
-  return dom.div({ className: "close-btn", onClick: handleClick },
+function CloseButton({ handleClick, buttonClass }) {
+  return dom.div({ className: buttonClass !== undefined ? buttonClass : "close-btn", onClick: handleClick },
```
Comment (jasonLaster) [inline public/js/components/CloseButton.js]: should be additive
checkout classnames https://github.com/devtools-html/debugger.html/blob/master/public/js/components/Breakpoints.js#L84-L88

**Comment by jasonLaster** [comment]
Thanks @samuelfullerthomas! 

I noticed a couple of linting things were off, but that should be easy to catch with `npm run lint`

**Comment by samuelfullerthomas** [comment]
Cool! I've updated @jasonLaster

**Pull request merged**